tests/project/test_config.py

```diff
@@ -0,0 +1,17 @@
+import os
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from unittest import mock
+
+from snowcli.cli.project.config import load_project_config, load_local_config
+
+from tests.project.fixtures import *
+
+def test_project_1_config(with_project_1):
```

Comment:
I assume we'll add the fixture as decorator with parameters for different test inputs later on, instead of passing it as an argument.

Reply:
Yeah, we can use the same `parameterize` trick later.